app/views/shared/_location_map.html.erb

```diff
@@ -1,6 +1,6 @@
 <my-map
   style="width: 100%; height: 600px;"
-  osVectorTilesApiKey='<%= Rails.configuration.os_vector_tiles_api_key %>'
+  osProxyEndpoint="<%= os_proxy_endpoint %>"
```

Comment:
Can we use the route helper for this instead? If we have `as: 'os_proxy'` in the route definition I think we can refer to it as `os_proxy_url`.